docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -69,4 +69,13 @@ $ kubectl get dataimportcron cron-test -o yaml | grep -B 5 storageClassName
         storageClassName: rook-ceph-block
 ```
 
+### Disconnected Environment
+
+For a cluster installed in a disconnected environment, the default golden images can't be polled because they are out of reach, thus the alert will be fired.  
+In that case, it is encouraged to disable the feature gate `enableCommonBootImageImport` to opt-out from automatic updates of the common data import cron templates and refrain from this alert being fired.  
+This can be done by patching the HyperConverged CR:
+```bash
+$ kubectl patch hco kubevirt-hyperconverged -n $CDI_NAMESPACE --type json -p '[{"op": "replace", "path": "/spec/featureGates/enableCommonBootImageImport", "value": false}]' 
```

Comment:
```suggestion
$ kubectl patch hco kubevirt-hyperconverged -n $HCO_NAMESPACE --type json -p '[{"op": "replace", "path": "/spec/featureGates/enableCommonBootImageImport", "value": false}]' 
```

Reply:
I wanted to reuse the same env var used earlier in the page, and avoid introducing another env var, which has the same value. HCO and CDI are always in the same namespace.
But I'll change as for your suggestion

Reply:
no no, if you have a good reason, keep it.
I htought it was a typo.